src/pages/header/header-page/header-page.js

```diff
@@ -22,7 +22,8 @@ const pathToHeaderAddPage = routes.pathToAddHeader;
 const tableTitles = config.tableHeadRowTitles.headers;
 
 const HeaderPage = () => {
-  const styles = useStyles();
+  const common = useCommonStyles();
```

Comment:
Can you rename this `common` variable to styles or classes?

Reply:
fixed